templates/head.html

```diff
@@ -11,6 +11,13 @@
 {{^includesIframe}}
 <script async custom-element="amp-iframe" src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"></script>
 {{/includesIframe}}
+{{^includesConsent}}
```

Comment:
Can you make both imports depend on the availability:

```
{{^includesGeo}}
{{^includesConsent}}
<script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
<script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>    
{{/includesConsent}}
{{/includesGeo}}
```

Reply:
I wouId have thought that would cause problems for pages that just used <amp-geo>.  But I've done it!